app/models/katello/deb.rb

```diff
@@ -0,0 +1,89 @@
+module Katello
+  class Deb < Katello::Model
+    include Concerns::PulpDatabaseUnit
+
+    CONTENT_TYPE = Pulp::Deb::CONTENT_TYPE
+
+    has_many :repositories, :through => :repository_debs, :class_name => "Katello::Repository"
+    has_many :repository_debs, :class_name => "Katello::RepositoryDeb", :dependent => :destroy, :inverse_of => :deb
+
+    has_many :content_facets, :through => :content_facet_applicable_debs, :class_name => "Katello::Host::ContentFacet"
```

Comment:
I don't think this applicable_debs piece is useful for debs right now.  This requires special support in pulp to identify that some deb can be upgraded on a particular host, by knowing a) what is installed, b) what repositories a system is using ,and c) what deps are in that repo.  This is only supported in pulp for rpms and errata AFAIK.  Does the deb plugin add this for debs?   If it does then i'd expect to see this data being imported elsewhere in the PR, but i don't think i see that.

Reply:
Sounds reasonable. I never really understood the content-facet thing. So this one is owed to copying from RPM. Is there a good explanation lying around somewhere?

Reply:
Some pulp documentation is here:  https://docs.pulpproject.org/dev-guide/integration/rest-api/consumer/applicability.html

but basically it lets you see what is available to a system to be upgraded.  I think it would be a great to have in the future for deb, but i'm assuming its not currently available in pulp_deb